src/sentry/api/endpoints/sentry_app_details.py

```diff
@@ -41,6 +41,15 @@ def put(self, request, sentry_app):
             'overview': request.json_body.get('overview'),
         }
 
+        if data.get('events'):
+            if ('error' in data['events']) and not features.has('organizations:integrations-event-hooks',
+                                                                sentry_app.owner,
+                                                                actor=request.user):
+
+                return Response({"non_field_errors": [
+                    "Your organization does not have access to the 'error' resource subscription.",
```

Comment:
@MeredithAnya How do we know that this is the error? What if the error was from something else?

Reply:
`'error'`is a valid event type for the integration platform, but it's feature gated behind `integrations-event-hooks` right now. So if someone is trying to update their app to have the `error.created` functionality then they must have that flag enabled for their org. 

Is that what you were looking for? Or is there something else I'm missing ?

Reply:
@MeredithAnya That makes sense. Thanks for the explanation.